src/Notifier.cpp

```diff
@@ -61,6 +63,24 @@ bool Notifier::Initialize(const size_t handshakeCount) {
 }
 
 /*
+ * Receives message data without caring about handshakes, added sleeping code
+ * to prevent pounding the cpu
+ */
+std::string Notifier::ReceiveData() {
+   StopWatch waitCheck;
+   while (waitCheck.ElapsedSec() < mMaxTimeoutInSec && mKeepRunning.load()) {
+      std::string msg;
+      if (gHandshakeQueue->GetShot(msg, mGetShotTimeoutInMs)) {
```

Comment:
```mGetShotTimeoutInMS``` should just be 0 in this case. We want to just check right away and then sleep if there wasn't anything.